plugins/20_events/hook

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+[[ ! "$DOKKU_EVENTS" ]] || dokku_log_pluginhook_call "$(basename $0)" "$@"
```

Comment:
On Wed, Jul 1, 2015 at 6:50 PM, Michael Hobbs <notifications@github.com> wrote:
> This question may be naive, but any reason we're doing a double negative here. More directly why not do this?

It's just my personal preference, but if that breaks the code style
I'm more than happy to fix it then.


-- 
Alessio Treglia          | www.alessiotreglia.com
Debian Developer         |     alessio@debian.org
Ubuntu Core Developer    |  quadrispro@ubuntu.com
0416 0004 A827 6E40 BB98 90FB E8A4 8AE5 311D 765A